src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/DynamicTableViewCell.swift

```diff
@@ -36,10 +36,31 @@ struct DynamicCell {
 extension DynamicCell {
 	enum CellReuseIdentifier: String, TableViewCellReuseIdentifiers {
 		case dynamicTypeText = "regularCell"
+		case dynamicTypeTextView = "textViewCell"
```

Comment:
To be consistent here: Can we refactor `dynamicTypeText` to `dynamicTypeLabel`. The raw value should also be adjust accordingly.